src/angular/typography.md

```diff
@@ -0,0 +1 @@
+## Typography doc test
```

Comment:
We will probably don't have a separate typography file, but only the getting started. but can still discuss :)